### PR TITLE
#24 Use SVG icons instead of font awesome

### PR DESCRIPTION
**Pull request by SergKam**
I downloaded only SVG icons we use and added them to CSS as icon background.

The uncompressed JS bundle size (compiled by elm v0.19) dropped from 1.328 MB (2.393 MB in elm v0.18) to only 0.678 MB.
In CSS it is only +10Kb

### CODE REVIEW

File: client/assets/styles.css [hunk 1]
```diff
@@ -967,3 +968,68 @@ body {
...
+}
+
+[disabled] > .icon,
+.icon.icon--disabled {
+    transform: none;
+}
+
+/*
+SVG icons was downloaded from https://fontawesome.com/icons
+TODO You must give appropriate credit,
```
Comment (lmontrieux) [inline client/assets/styles.css]: What is considered 'appropriate credit'?

Comment (SergKam) [reply client/assets/styles.css]: Oh, actually nothing. https://fontawesome.com/license/free


> Attribution
> Attribution is required by MIT, SIL OFL, and CC BY licenses. Downloaded Font Awesome Free files already contain embedded comments with sufficient attribution, so **you shouldn't need to do anything additional** when using these files normally.
> 
> We've kept attribution comments terse, so we ask that you do not actively work to remove them from files, especially code. They're a great way for folks to learn about Font Awesome.
>

Comment (SergKam) [reply client/assets/styles.css]: I will remove TODO then

**Comment by ferbncode** [comment]
:+1:

**Comment by SergKam** [comment]
:+1:

**Pull request merged**